start/src/main/java/org/apache/accumulo/start/classloader/vfs/ContextManager.java

```diff
@@ -19,95 +19,96 @@
 package org.apache.accumulo.start.classloader.vfs;
 
 import java.io.IOException;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.net.URLClassLoader;
+import java.util.Arrays;
 import java.util.HashMap;
 import java.util.Map;
 import java.util.Map.Entry;
 import java.util.Set;
 import java.util.function.Supplier;
+import java.util.stream.Collectors;
 
-import org.apache.commons.vfs2.FileSystemException;
-import org.apache.commons.vfs2.FileSystemManager;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-@Deprecated
-class ContextManager {
+//@Deprecated
+public class ContextManager {
 
   private static final Logger log = LoggerFactory.getLogger(ContextManager.class);
 
+  public static final String CONTEXT_CLASSPATH_PROPERTY = "general.vfs.context.classpath.";
+
   // there is a lock per context so that one context can initialize w/o blocking another context
   private class Context {
-    AccumuloReloadingVFSClassLoader loader;
+    URLClassLoader loader;
     ContextConfig cconfig;
     boolean closed = false;
 
     Context(ContextConfig cconfig) {
       this.cconfig = cconfig;
     }
 
-    synchronized ClassLoader getClassLoader() throws FileSystemException {
+    synchronized ClassLoader getClassLoader() throws IOException {
       if (closed) {
         return null;
       }
 
       if (loader == null) {
-        log.debug(
-            "ClassLoader not created for context {}, creating new one. uris: {}, preDelegation: {}",
-            cconfig.name, cconfig.uris, cconfig.preDelegation);
-        loader =
-            new AccumuloReloadingVFSClassLoader(cconfig.uris, vfs, parent, cconfig.preDelegation);
+        log.debug("ClassLoader not created for context {}, creating new one. uris: {}",
+            cconfig.name, cconfig.uris);
+        loader = new URLClassLoader(Arrays.stream(cconfig.uris.split(",")).map(url -> {
+          try {
+            return new URL("file://" + url);
+          } catch (MalformedURLException e) {
+            throw new RuntimeException(e);
+          }
+        }).collect(Collectors.toList()).toArray(new URL[] {}), parent);
       }
 
-      return loader.getClassLoader();
+      return loader;
     }
 
     synchronized void close() {
       closed = true;
-      if (loader != null) {
-        loader.close();
-      }
       loader = null;
     }
   }
 
   private Map<String,Context> contexts = new HashMap<>();
 
   private volatile ContextsConfig config;
-  private FileSystemManager vfs;
-  private ReloadingClassLoader parent;
+  private ClassLoader parent;
 
-  ContextManager(FileSystemManager vfs, ReloadingClassLoader parent) {
-    this.vfs = vfs;
+  public ContextManager(ClassLoader parent) {
     this.parent = parent;
```

Comment:
Not sure we need to the parent argument, the parent classloader will be the ClassLoader for this class.